apps/fhir/bluebutton/loggers.py

```diff
@@ -1,11 +1,10 @@
-import json
-import logging
+import apps.logging.request_logger as logging
 
 
 """
   Logger and logging function for waffle flags, switches
 """
-waffle_event_logger = logging.getLogger('audit.waffle.event')
+waffle_event_logger = logging.getLogger(logging.AUDIT_WAFFLE_EVENT_LOGGER)
 
 
 def log_v2_blocked(user=None, path=None, app=None, err=None, **kwargs):

```

Comment:
We basically stopped using this function above didn't we? Maybe it's no longer necessary?